### DISCUSSION TITLE
onc proxy

### DISCUSSION THREAD

**Discussion by xoidex** [3kh0/ext-remover#959] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
is it possible to change your proxy settings with a onc file?
and if so, does anyone have a example onc file i can edit?

ty

**Comment by PSYclownYT** [comment]
read the readme (yes you need access to chrome:network-internals)

**Reply by xoidex** [reply]
what do i read?
???

**Reply by xoidex** [reply]
@PSYclownYT chrome://network-internals isn't even mentioned in the readme
do you mean chrome://network?

**Reply by xoidex** [reply]
@PSYclownYT dude you gonna respond or what

**Reply by AshtonDavies** [reply]
> @PSYclownYT chrome://network-internals isn't even mentioned in the readme
> do you mean chrome://network?

I think he means chrome://network.

**Reply by xoidex** [reply]
@PSYclownYT still, the question isn't how to import an onc, it's if you can import one (with proxy settings.) you're not helpful at all.

**Comment by 1coolmatt8083** [comment]
yo has any got a proxy site i can use with light speed agent

**Reply by xoidex** [reply]
(pick a letter a to i).cpfrx.info might work

**Reply by 1coolmatt8083** [reply]
yo ok well it dident
![Screenshot 2024-03-21 7 53 22 AM](https://github.com/3kh0/ext-remover/assets/162379962/ffa72ec0-2c93-4713-ae66-d0d5f1bca825)

**Reply by AshtonDavies** [reply]
> yo ok well it dident
> ![Screenshot 2024-03-21 7 53 22 AM](https://github.com/3kh0/ext-remover/assets/162379962/ffa72ec0-2c93-4713-ae66-d0d5f1bca825)
> 

Subdomain >> a. << at the beginning.

**Reply by 1coolmatt8083** [reply]
oh opps

**Comment by 1coolmatt8083** [comment]
does any one have a way or proxy or somthin i can use to bypass the light speed agent.(ex i would like one that i can play roblox with)

**Reply by 1coolmatt8083** [reply]
or like a browser or somthin

**Reply by 1coolmatt8083** [reply]
OR SOMTHiN BRO DOES ANY ONE READ THIS COMMENT ANY MORE

**Reply by xoidex** [reply]
dude shut up
you don't even need a proxy to play roblox on chromebooks
why do you even want to play roblox?
if you want proxies go deploy your own, stop commenting here

**Reply by 1coolmatt8083** [reply]
yo are u kinding me im trying to be nice bro but ok if u want to be that way i just have to kill ur grama hehehehehhehehehehehhheheheheheeheheeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeehhhhhhhhhhhhhhheeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee arrr grama killinngggg timmmmeee he ehe hehe hehe e heh ee arrggggggggg I will kill u too ooo yeeee hy=eeee ejejwhjfnehbfjefhbken fuckk u u little dumb bich mother fucker niger  bich fucking hord retart nimrod son of a bich fucker suck a pusy mother bich niger let a fucker eat ur mother a fuck a niger in a pussy

**Reply by 1coolmatt8083** [reply]
on some chromebooks you do, retard

**Reply by 1coolmatt8083** [reply]
im so sorry i did not mean to say that 
# # # # # #so so so so sos os sos so sos os so sosos os o soso osso s sos so sORRRRRYYYYYYYY

**Reply by 1coolmatt8083** [reply]
NOT hehehhehehe

**Reply by 1coolmatt8083** [reply]
# FUCK THIS DISCUTUTION RETARD

**Reply by xoidex** [reply]
actual child lmfao

**Reply by 1coolmatt8083** [reply]
sorry bro acully i meant to say that i waqs trying to find a proxy that i can use for the light speed agent do u no any sites i can use or strats

**Reply by xoidex** [reply]
real bro 💀

**Reply by xoidex** [reply]
no way github marked my comment as spanish and then fixed my capatlization

**Reply by 1coolmatt8083** [reply]
![Screenshot 2024-03-28 9 13 37 AM](https://github.com/3kh0/ext-remover/assets/162379962/d1b144b2-3c5c-4005-9279-405578d4899a)

**Reply by 1coolmatt8083** [reply]
ya it says **TRANSLATE TO ENGLISH**

**Reply by 1coolmatt8083** [reply]
![Screenshot 2024-03-28 9 13 37 AM](https://github.com/3kh0/ext-remover/assets/162379962/72b12816-b1f9-462e-8e69-4e25d6afee9d)

**Reply by 1coolmatt8083** [reply]
have u seen the new game i made

**Reply by 1coolmatt8083** [reply]
https://github.com/1coolmatt8083/1coolmatt8083
its the karlson 3d one it took me a whole year to make it and i just finished it

**Comment by HackrTP** [comment]
> is it possible to change your proxy settings with a onc file? and if so, does anyone have a example onc file i can edit?
> 
> ty

Any update?

**Comment by S-PScripts** [comment]
closed unless you want an answer for yourself

it's best to ask in the discord server
